DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -348,6 +353,10 @@ CentreAnswersData centreAnswersData
         {
             var delegateUser = userDataService.GetDelegateUserById(editDelegateDetailsData.DelegateId);
             var (adminUser, delegateUsers) = GetUsersByEmailAddress(delegateUser!.EmailAddress!);
+            if (delegateUsers.Count == 0)
+            {
+                delegateUsers.Add(delegateUser);
+            }

```

Comment:
Is there a particular reason to do this this way, rather than bypassing the UpdateDelegateAccountDetails (line 373) call with no other delegate accounts? UpdateDelegate (line 380) updates the names and email on the single delegate anyway.